Marlin/stepper.cpp

```diff
@@ -601,14 +639,28 @@ void Stepper::init() {
   TCNT1 = 0;
   ENABLE_STEPPER_DRIVER_INTERRUPT();
 
-  #if ENABLED(ADVANCE)
+  #if ENABLED(ADVANCE) || ENABLED(LIN_ADVANCE)
+
+    #if ENABLED(LIN_ADVANCE)
+
+      for (int i = 0; i < EXTRUDERS; i++) {
```

Comment:
@thinkyhead shouldn't we clean up the "old" `ADVANCE` feature ? Having two advance features using two different methods but when only one has active support and is known to work.. I would cull the other one.